src/nifty_ls/__init__.py

```diff
@@ -5,6 +5,8 @@
 
 __all__ = ['lombscargle', 'NiftyResult']
 
+from .version import __version__ as __version__
```

Comment:
Could this be shorted to `from .version import __version__`?

Reply:
I believe so!

Reply:
It's a `ruff F:401 unused-import` error so not critical, but it does seem like the solution is to use a redundant alias `__version__ as __version__` to get around this problem.

[https://docs.astral.sh/ruff/rules/unused-import/](https://docs.astral.sh/ruff/rules/unused-import/)

Reply:
Adding it to `__all__` should also fix this!